### PR TITLE
OperatorScan - don't call onNext after onError is called

### PR DESCRIPTION
**Pull request by davidmoten**
title says it all

### CODE REVIEW

File: src/main/java/rx/internal/operators/OperatorScan.java [hunk 1]
```diff
@@ -104,6 +104,7 @@ public void onNext(T currentValue) {
                         this.value = accumulator.call(this.value, currentValue);
                     } catch (Throwable e) {
                         child.onError(OnErrorThrowable.addValueAsLastCause(e, currentValue));
+                        return;
```
Comment (zsxwing) [inline src/main/java/rx/internal/operators/OperatorScan.java]: Good catch. Could you also add `Exceptions.throwIfFatal(e);` above `child.onError` like `OperatorMap`?

**Comment by davidmoten** [comment]
Added ```Exceptions.throwIfFatal``` as suggested by @zsxwing

**Comment by akarnokd** [comment]
Thanks.

**Comment by benjchristensen** [comment]
:+1:

**Pull request merged**